collections/guides/quickstarts/quickstart_ruby.md

```diff
@@ -0,0 +1,222 @@
+---
+layout: default
+title: Quickstart with Ruby
+permalink: /getting_started/quickstart_with_ruby/
+categories: quickstart
+nav_order: 1
```

Comment:
Does nav_order need to be unique? Node.js and Ruby quickstarts have the same `nav_order: 1`.